cleverhans/utils.py

```diff
@@ -8,7 +8,10 @@
 import matplotlib.pyplot as plt
 from keras.models import Sequential
 from keras.layers import Dense, Dropout, Activation, Flatten
-from keras.layers import Convolution2D
+if int(keras.__version__[0]) >= 2:
```

Comment:
This check is brittle.

Should use distutils.version or possibly packaging.version.parse for a more robust check.

http://stackoverflow.com/questions/11887762/compare-version-strings-in-python